client/src/research/InteractionsView.js

```diff
@@ -1,15 +1,52 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-
+import db from '../../../tmp/swipe-right-db.json';
+import {InteractionTypes} from '../shared/data.js';
 
 // Render a list of logged user interactions
 class InteractionsView extends Component {
-  render() {
+  render() {  
+    // unpack!
+    //filter out testing data
+
+    const interactions = db.interactions.filter(row =>{  
+      if (row.session.workshopCode === 'foo') return false;
+      if (row.session.workshopCode === 'demo') return false;
+      if (row.session.workshopCode === 'code.org') return false;
+      if (row.session.workshopCode.indexOf('DEMO') === 0) return false;
+      if (row.session.identifier === 'UNKNOWN_IDENTIFIER') return false;
+      if (row.session.identifier === '') return false;
+      if (row.session.identifier === 'kevin') return false;
+      return true;
+    });
+    // show it!
+
+
     // TODO render these as a filterable table
     // 1. Render using a <pre> tag and JSON.stringify(json, null, 2) so see if things look right
     // 2. Change to a table with each interaction as a row using <table><tr><td> etc.
     // 3. Add https://github.com/bvaughn/react-virtualized and use that
-    return <div>TODO!</div>;
+    return( 
+      <div>
+        {this.renderPercentSwipeRight(interactions)}
+        <table> {interactions.map(row =>{
```

Comment:
The single whitespace after `<table> ` and ` </table>` might through things off, that happened when I tried to run this locally.  Can you remove those extra spaces?